### ISSUE TITLE
focus lost in certain cases

### ISSUE THREAD

**Issue by phazei** [fnagel/jquery-ui#22]
When done in dropdown style, or when list is completely above or below box, if you click and click the box again closing it, without selecting anything new, it loses focus.  After that you can't use the arrows unless you tab away and tab back.

**Comment by fnagel** [comment]
This bug is confirmed. Accessibility issue.

**Comment by fnagel** [comment]
Thanks for reporting. The problem was a missing parameter in this.newelement mousedown event. 

Fixed in latest commit.

**Comment by pdias** [comment]
Hello, i have this problem... but i'm using selecmenu within a dialog.

Ideas to solve?

Thanks,

Paulo

**Comment by fnagel** [comment]
Please open a new issue with reference to this one, see #61
